### DISCUSSION TITLE
[Unstable] Container tag character limit breaks autofill / bot-cleanup (and potential modded) tags

### DISCUSSION THREAD

**Discussion by Ek-Videogames** [FakeFishGames/Barotrauma#14550] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

So as it turns out... whatever changed that caused the '20 character limit' in hull names also affects container tags.

Most container tag names being so long-winded that you can really only fit one tag in, and any second tag added will almost always end up cutoff.

### Reproduction steps

![image](https://github.com/user-attachments/assets/72933f1a-59f0-44e5-9706-b298eb6e9809)

![image](https://github.com/user-attachments/assets/c6683070-888f-44e8-822d-259c6b6dd74b)

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens outside single player or multiplayer game modes (e.g. game launches on startup, something broken in the main menu)

### Version

v1.6.4.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

### More Information

Reasons why you would want to give a container multiple tags besides for autofill:
- Bot cleanup 'sorting' behavior
- Modded autofill or event-related tags

**Comment by Regalis11** [comment]
This has been fixed in our private repo now, fix coming up in the next unstable build. Sorry for the inconvenience!
